src/current/v25.2/detect-hotspots.md

```diff
@@ -0,0 +1,141 @@
+  - If **No**, check for a node outlier in [CPU percent](#b-cpu-percent) metric. 
+
+### B. CPU percent
+
+- Navigate to the DB Console **Metrics** page **Hardware** dashboard.
+- Monitor the [**CPU Percent** graph]({% link {{ page.version.version }}/ui-hardware-dashboard.md %}#cpu-percent).
+- CPU usage typically increases with traffic volume.
+- Check if the CPU usage of the hottest node is 20% or more above the cluster average. For example, node `n5`, represented by the green line in the following **CPU Percent** graph, hovers at around 87% at time 17:35 compared to other nodes that hover around 20% to 25%.
+  
+<img src="{{ 'images/v25.2/detect-hotspots-cpu-percent.png' | relative_url }}" alt="graph of CPU Percent utilization per node showing hot key" style="border:1px solid #eee;max-width:100%" />
+
+- Is there a node with a maximum value that is a clear outlier in the cluster for the CPU percent metric?
+
+  - If **Yes**, note the ID of the [hot node]({% link {{ page.version.version }}/understand-hotspots.md %}#hot-node) and the time period when it was hot. Proceed to check for a [`popular key detected `log](#a-popular-key-detected).
+  - If **No**, and the metrics outlier appears in a metric other than CPU percent or latch conflict wait duration, consider causes other than a hotspot.
+
+## Step 2. Check for existence of `no split key found` log
+
+The [`no split key found` log]({% link {{ page.version.version }}/load-based-splitting.md %}#monitor-load-based-splitting) is emitted in the [`KV_DISTRIBUTION` log channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels). This log is only emitted when a single replica begins using a significant percentage of the resources on the node where it resides.
+
+This log is not associated with a specific event type but includes an unstructured message, for example:
+
+```
+I250523 21:59:25.755283 31560 13@kv/kvserver/split/decider.go:298 ⋮ [T1,Vsystem,n5,s5,r1115/3:‹/Table/106/1/{113338-899841…}›] 2979  no split key found: insufficient counters = 0, imbalance = 20, most popular key occurs in 36% of samples, access balance right-biased 98%, popular key detected, clear direction detected
+```
+
+In the preceding log example, the square-bracketed tag section provides the following information:
+
+- node ID: `n5` indicates the node ID is 5.
+- range ID: `r1115` indicates the range ID is 1115.
+
+The timestamp at the beginning of the log is `250523 21:59:25.755283`.
+
+The unstructured message ends with one of the following string combinations:
+
+1. `popular key detected, clear direction detected`
+1. `popular key detected, no clear direction`
+1. `no popular key, clear direction detected`
+1. `no popular key, no clear direction`
+
+### A. `popular key detected`
+
+The `popular key detected` log indicates that a significant percentage of reads or writes target a single row within a range.
```

Comment:
suggest linking "range" to https://www.cockroachlabs.com/docs/v25.2/architecture/glossary.html#range since the user needs to understand replicas/ranges pretty well to troubleshoot this